### PR TITLE
[SOL-2190] Fix basket alert message close button position.

### PR DESCRIPTION
**Pull request by vkaracic**
https://openedx.atlassian.net/browse/SOL-2190

Here's the close button now:
screenshot from 2017-01-16 10-32-20

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vkaracic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?number=1094&repo=edx%2Fecommerce"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Comment by vkaracic** [comment]
@mjfrey please review.

**Comment by vkaracic** [comment]
@schenedx if you have time, could you please review this?

File: ecommerce/templates/oscar/partials/alert_messages.html [hunk 1]
```diff
@@ -33,7 +33,9 @@
                     {% endif %}
                 {% endif %}
             </div>
-            <a class="close" data-dismiss="alert" href="#">&times;</a>
+            <div class="close-btn col-sm-1 col-xs-2">
+                <a class="close" data-dismiss="alert" href="#">&times;</a>
```
Comment (schenedx) [inline ecommerce/templates/oscar/partials/alert_messages.html:37]: If you are not relying on this class `close`, please remove it.

**Review by schenedx — APPROVED** [review]
👍

File: ecommerce/templates/oscar/partials/alert_messages.html (hunk 1 above)
Comment (vkaracic) [reply ecommerce/templates/oscar/partials/alert_messages.html:37]: There are CSS rules for that class that we inherit from Oscar.

**Pull request merged**